gui/src/main/java/org/jboss/as/console/client/rbac/SecurityContextImpl.java

```diff
@@ -97,6 +97,27 @@ private AuthorisationDecision checkPriviledge(Priviledge p, boolean includeOptio
     }
 
     /**
+     *     Check to see if an exception exists in childContexts for this address,
+     *     or an equivalent address i.e. an actual group name if '*' is specified
+     */
+    private boolean exceptionRuleExists(String address) {
+        if (address.contains("{addressable.group}")) {
+            String[] addressSplit = address.split("/");
+            String addressDeployment = addressSplit[addressSplit.length - 1];
+
+            for (Map.Entry<String, SecurityContext> entry : childContexts.entrySet()) {
```

Comment:
Can you confirm that your fix works only when there are deployment sub resources? See https://issues.jboss.org/browse/WFLY-4976